src/Routes/V1/Shop/customer-routes.php

```diff
@@ -80,7 +80,7 @@
 
     Route::post('customer/cart/add/{productId}', [CartController::class, 'add']);
 
-    Route::put('customer/cart/update', [CartController::class, 'update']);
+    Route::post('customer/cart/update', [CartController::class, 'update']);
```

Comment:
Don't change the method. For updating PUT method will be used.

Reply:
Fixed